packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -387,6 +398,7 @@ function ActiveRequests({ votingAccount, votingGateway, snapshotContract }) {
       commits.push({
         identifier: pendingRequests[index].identifier,
         time: pendingRequests[index].time,
+        ancillaryData: "0x",
```

Comment:
So clearly the committed `ancillaryData` and the revealed `ancillaryData` have to be identical. Furthermore, in the future we might want to change this field dynamically depending on the type of vote we're having. Therefore I think its sensible to set a constant ANCILLARY_DATA object at the file level that both this and reveals can read from. This way we can more easily upgrade the ancillary data logic to be dynamic based on types of votes.

Reply:
The next PR will be to add a field so the user can change ancillary data, but yea the default should at least be a constant.